sdk/python/tests/compiler/testdata/tekton_pipeline_conf.yaml

```diff
@@ -48,12 +48,22 @@ spec:
           - sh
           - -c
           image: busybox
+        workspaces:
+        - name: new-ws
+          readOnly: true
         metadata:
           labels:
             pipelines.kubeflow.org/cache_enabled: "true"
           annotations:
+            workspaces: '{"new-ws": {"readOnly": true}}'
```

Comment:
do we need this annotation in the output yaml?

Reply:
thanks, I updated the code to remove this annotation.